src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanel.java

```diff
@@ -96,7 +97,8 @@ private void triggerOnChange() {
     onChangeCallbacks.forEach(Runnable::run);
   }
 
-  public void setEnabled(boolean enabled) {
+  public void setEnabled(boolean enabled, boolean savePasswordsConsent) {
```

Comment:
Does this method now do more than setting an enabled state?

Reply:
It just needs more information to set the enabled state. When the savePasswordsConsent is `false`, the remember password checkbox must be disabled.

I use the `setEnabled` method because it's present in all `JComponents` and it felt right, but in this case, it's more like a `lockUI` that prevents/enables user interaction.